### ISSUE TITLE
Errors when using Map as a context

### ISSUE THREAD

**Issue by ynaciri** [verronpro/docx-stamper#109]
**Describe the bug**
On **v1.4.7** release, when using a Map as a context, storing keys for each placeholders mapped to values and MapAccessor to resolve them, DocxStamper fails to do so.

**To Reproduce**
1. Add MapAccessor in the configuration using `DocxStamperConfiguration.setEvaluationContextConfigurer(context -> context.addPropertyAccessor(new MapAccessor()))`
2. Set FailOnUnresolvedExpression to false in the configuration
3. Prepare a map for the context with a property inside an object inside a list
4. Create a docx template with a placeholder inside a repeatParagraph using the previous property
6. Stamp the docx template
7. See warnings in the console like : `Expression ${xxx} could not be resolved against context root of type class javassist.util.proxy.java_util_HashMap_$$_jvstd68_0. Reason: EL1021E: A problem occurred whilst attempting to access the property 'xxx': 'null'. Set log level to TRACE to view Stacktrace.`
8. An exception is thrown (see below)

**Example:** check the MapAccessorAndReflectivePropertyAccessorTest

**Expected behavior**
The exception should be catched and logged without interrupting the document stamping.

**Desktop (please complete the following information):**
 - docx-stamper version 1.4.7

**Similar issues:**
#54 Proxy-ing problems

**Stacktrace**
```
org.springframework.expression.spel.SpelEvaluationException: EL1008E: Property or field 'xxx' cannot be found on object of type 'javassist.util.proxy.java_util_HashMap_$$_jvst116_0' - maybe not public or not valid?
	at org.springframework.expression.spel.ast.PropertyOrFieldReference.readProperty(PropertyOrFieldReference.java:217) ~[spring-expression-5.3.6.jar:5.3.6]
	at org.springframework.expression.spel.ast.PropertyOrFieldReference.getValueInternal(PropertyOrFieldReference.java:104) ~[spring-expression-5.3.6.jar:5.3.6]
	at org.springframework.expression.spel.ast.PropertyOrFieldReference.getValueInternal(PropertyOrFieldReference.java:91) ~[spring-expression-5.3.6.jar:5.3.6]
	at org.springframework.expression.spel.ast.SpelNodeImpl.getValue(SpelNodeImpl.java:112) ~[spring-expression-5.3.6.jar:5.3.6]
	at org.springframework.expression.spel.standard.SpelExpression.getValue(SpelExpression.java:272) ~[spring-expression-5.3.6.jar:5.3.6]
	at org.wickedsource.docxstamper.el.ExpressionResolver.resolveExpression(ExpressionResolver.java:40) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.replace.PlaceholderReplacer.resolveExpressionsForParagraph(PlaceholderReplacer.java:119) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.replace.PlaceholderReplacer$1.onParagraph(PlaceholderReplacer.java:104) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.util.walk.CoordinatesWalker.walkTableCell(CoordinatesWalker.java:130) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.util.walk.CoordinatesWalker.walkTableRow(CoordinatesWalker.java:117) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.util.walk.CoordinatesWalker.walkTable(CoordinatesWalker.java:103) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.util.walk.CoordinatesWalker.walkContent(CoordinatesWalker.java:77) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.util.walk.CoordinatesWalker.walk(CoordinatesWalker.java:38) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.replace.PlaceholderReplacer.resolveExpressions(PlaceholderReplacer.java:107) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.DocxStamper.replaceExpressions(DocxStamper.java:167) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.DocxStamper.stamp(DocxStamper.java:142) ~[docx-stamper-1.4.7.jar:na]
	at org.wickedsource.docxstamper.DocxStamper.stamp(DocxStamper.java:121) ~[docx-stamper-1.4.7.jar:na]
```

**Comment by ynaciri** [comment]
After fixing the issue with repeatParagraph ignoring the FailOnUnresolvedPlaceholder, the value property is replaced by the toString representation in the container object :

![image](https://user-images.githubusercontent.com/8029435/211050468-dc46b0dd-2068-4ec0-88d8-5343aabe1ede.png)

**Comment by ynaciri** [comment]
After fixing the issue with the ignored failOnUnresolvedExpression, the docx is generated without replacing the expressions displayed in warnings.

**Comment by ynaciri** [comment]
fixed on master-javax since v1.5.0-javax

**Comment by ynaciri** [comment]
solved on master by #129
